pages/create.unit.page.ts

```diff
@@ -0,0 +1,554 @@
+        announcementNameInput = this.page.locator('input[data-testid="custom-input"]').first();
+        announcementNameInputError = this.page.locator('[data-testid="descriptionError"]').first();
+        vehicleManufacturerTitle = this.page.locator('div[class*="SelectManufacturer_title"]');
+        vehicleManufacturerInputContainer = this.page.locator('div[class*="CustomSelectWithSearch_searchInput"]');
+        vehicleManufacturerInput = this.page.locator('input[data-testid="input-customSelectWithSearch"]');
+        vehicleManufacturerInputError = this.page.locator('div[class*="CustomSelectWithSearch_errorTextVisible"]');
+        vehicleManufacturerInputSearchIcon = this.page.locator('div[class*="CustomSelectWithSearch_searchInput"] > svg');
+        vehicleManifacturerDropDown = this.page.locator('div[class*="CustomSelectWithSearch_searchedServicesCat_wrapper"]');
+        vehicleManufacturerDropDownOption = this.page.locator('div[class*="CustomSelectWithSearch_flexForServices"]');
+        optionNotFoundMessage = this.page.locator('[data-testid="p2-notFound-addNewItem"]');
+        selectedOptionInDropdown = this.page.locator('[class*="CustomSelectWithSearch_serviceText"]');
+        clearVehicleManifacturerDropDownIcon = this.page.locator('button[class*="CustomSelectWithSearch_serviceBtn"]');
+        modelNameTitle = this.page.locator('div[class*="CustomInput_title"]').getByText('Назва моделі');
+        modelNameInput = this.page.locator('input[data-testid="custom-input"]').nth(1);
+        modelNameInputError = this.page.locator('[data-testid="descriptionError"]');
+        technicalInfoTitle = this.page.locator('div[class*="CustomTextAriaDescription_title"]').first();
+        technicalInfoInput = this.page.locator('[data-testid="textarea-customTextAriaDescription"]').first();
+        descriptionInfoTitle = this.page.locator('div[class*="CustomTextAriaDescription_title"]').nth(1);
+        descriptionInfoInput = this.page.locator('[data-testid="textarea-customTextAriaDescription"]').nth(1);
+        addressSelectionTitle = this.page.locator('[class*="AddressSelectionBlock_title"]');
+        addressSelectionInput = this.page.locator('[data-testid="mapLabel"]');
+        addressSelectionInputError = this.page.locator('[class*="AddressSelectionBlock_errorTextVisible"]');
+        selectOnMapBtn = this.page.locator('[class*="AddressSelectionBlock_locationBtn"]');
+        mapPopUp = this.page.locator('[data-testid="div-mapPopup"]');
+        mapPopUpTitle = this.page.locator('[class*="MapPopup_title"]');
+        mapPopUpCloseBtn = this.page.locator('[class*="MapPopup_title"]');
+        addressLine = this.page.locator('[data-testid="address"]');
+        mapPopUpSubmitBtn = this.page.locator('[class*="ItemButtons_darkBlueBtn"]');
+        mapContainer = this.page.locator('#map');
+        cancelBtn = this.page.locator('[data-testid="prevButton"]');
```

Comment:
There are existing cases that a page can be reloaded in its DOM tree, because of this unique IDs that the driver creates for each element will change too, which can lead to test failures (because the element with ID that was found at the start of test no longer exists).
So, here and further, you better use functions. See here an example [https://knowledge-base.luxequality.com/docs/traineeProgram/common_mistakes](url)